src/cript/nodes/core.py

```diff
@@ -296,6 +300,26 @@ class ReturnTuple:
         previous_condense_to_uuid = copy.deepcopy(NodeEncoder.condense_to_uuid)
         NodeEncoder.condense_to_uuid = condense_to_uuid
 
+        # For validation of just a node and its direct children, we enable a new mode in Node encoding
+        # For this node we explicitly list the nodes that are fully expressed, every other node is a UUID
+        previous_only_not_uuid: Optional[list[str]] = NodeEncoder.only_not_uuid
```

Comment:
is it possible to make the variable names more self documenting?

Reply:
Not sure, what do you have in mind?

Reply:
I think it works, if we can refactor later if needed